content/en/events/2021/kcc2021/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Contributor Celebration"
```

Comment:
We can drop 2021 from the directory path. Path should look like this
`content/en/events/2021/kcc2021/_index.md` -> `content/en/events/2021/kcc/_index.md`